alembic/versions/375dc804e56c_checked_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Checked migration
+
+Revision ID: 375dc804e56c
+Revises: 3c66b72cc8c3
+Create Date: 2024-10-31 15:09:41.897307
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '375dc804e56c'
+down_revision: Union[str, None] = '3c66b72cc8c3'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is also empty. Ensure you add the necessary operations to reverse the changes made in the `upgrade` function, allowing for a proper rollback if needed.